mesh/mesh.hpp

```diff
@@ -298,6 +298,11 @@ class Mesh
    // (true) is set in mesh_readers.cpp.
    static bool remove_unused_vertices;
 
+#ifdef MFEM_USE_NETCDF
```

Comment:
It would be better to group this much further down in the file near the `PrintXG`, `PrintVTK`, and similar member functions. The precise location is not important but somewhere within the Doxygen group `Print/Save/Export methods` would be appropriate.

Also, the name `PrintExodusII` would follow the present pattern.